### DISCUSSION TITLE
Far2L в Astra Linux SE

### DISCUSSION THREAD

**Discussion by VadimK60** [elfmz/far2l#2547] [Q&A]
Есть сведения, что сейчас рассматривается вопрос добавления всеми нами любимого файлового менеджера в Astra Linux.
В дистрибутив или основной репозиторий -- пока не известно.
Может вообще никуда не добавят... :(

Поэтому если:
* вам нравится идея интеграции Far2L в Astra Linux;
* вы представитель организации, в которой развёртывается Astra Linux,

то можете отправить свой запрос (на добавление программы) в техническую поддержку Астры через личный кабинет (https://lk-new.astralinux.ru). Возможно, это окажет положительный результат на исход рассмотрения.

К сожалению, обычные пользователи лишены возможности обращаться в техподдержку. :(
Но вы можете поучаствовать в обсуждении данного вопроса на канале, где присутствуют в том числе разработчики Астры (https://t.me/astralinux_chat/710562/1033682)

**Comment by VadimK60** [comment]
https://t.me/astralinux_chat/710562/1037875

Разработчики Астры отказались добавлять Far2L в свою ОС:
"_На данный момент мы не можем рассмотреть добавление данной утилиты. Данная операция будет доступна только после стабилизации Far2L в открытом сообществе_".

Реальная же причина отказа скорее всего в том, что мой запрос на добавление Far2L в Astra Linux оказался единственным. :(

**Comment by Mischutka** [comment]
Добрый день!

Я в своё время задавал подобный вопрос на уже закрытом форуме Astra Linux: https://forum.astralinux.ru/

Ответ был в духе "в обозримом будущем включение far2l в состав дистрибутива не рассматривается".
